interleave-in-parent.md

```diff
@@ -0,0 +1,172 @@
+
+### Interleave Prefix
+
+For interleaved tables to have Primary Keys that can be matched, the child table must use the parent table's entire Primary Key as a prefix of its own Primary Key––these matching columns are referred to as the **interleave prefix**. It's easiest to think of these columns as representing the same data, which is usually implemented with Foreign Keys.
+
+{{site.data.alerts.callout_success}}To formally enforce the relationship between each table's interleave prefix columns, we recommend using <a href="constraints.html#foreign-keys">Foreign Key constraints</a>.{{site.data.alerts.end}}
+
+For example, if you want to interleave `orders` into `customers` and the Primary Key of customers is `id`, you need to create a column representing `customers.id` as the first column in the Primary Key of `orders`&mdash;e.g., with a column called `customer`. So the data representing `customers.id` is the interleave prefix, which exists in the `orders` table as the `customer` column.
+
+### Key-Value Structure
+
+When you write data into the child table, it is inserted into the key-value store immediately after the parent table's key matching the interleave prefix.
+
+For example, if you interleave `orders` into `customers`, the `orders` data is written directly within the `customers` table in the key-value store. The following is a crude, illustrative example of what the keys would look like in this structure:
+
+~~~
+/customers/1
+/customers/1/orders/1000
+/customers/1/orders/1002
+/customers/2
+/customers/2/orders/1001
+/customers/2/orders/1003
+...
+/customers/n/
+/customers/n/orders/<order belonging to n>
+~~~
+
+By writing data in this way, related data is more likely to remain on the same key-value range, which can make it much faster to read from and write to. Using the above example, all of customer 1's data is going to be written to the same range, including its representation in both the `customers` and `orders` tables.
+
+## When should I interleave tables?
+
+You're most likely to benefit from interleaved tables when:
+
+  - Your tables form a [hierarchy](#interleaved-hierarchy)
+  - Queries maximize the [benefits of interleaving](#benefits)
+  - Queries do not suffer too greatly from interleaving's [tradeoffs](#tradeoffs)
+
+### Interleaved Hierarchy
+
+Interleaved tables typically work best when the tables form a hierarchy. For example, you could interleave the table `orders` (as the child) into the table `customers` (as the parent, which represents the people who placed the orders). You can extend this example by also interleaving the tables `invoices` (as a child) and `packages` (as a child) into `orders` (as the parent).
+
+The entire set of these relationships is referred to as the **interleaved hierarchy**, which are related through the interleave prefix columns.
```

Comment:
Grammatically, "which are related" directly modifies "interleaved hierarchy", which is awkward due: "are" doesn't match the singular "hierarchy". Maybe something like:

> The entire set of these relationships is referred to as the **interleave hierarchy** and is based on common interleave prefix columns.

Also, might help to link "interleave prefix" up to the corresponding section. 

Alternately, I think you could just leave out that final phrase, since the mechanics are explained earlier.